google/cloud/aiplatform/utils.py

```diff
@@ -318,14 +320,14 @@ def __init__(
         ):
             """Stores parameters needed to instantiate client.
 
-            client_class (AiPlatformServiceClient):
-                Required. Class of the client to use.
-            client_options (client_options.ClientOptions):
-                Required. Client options to pass to client.
-            client_info (gapic_v1.client_info.ClientInfo):
-                Required. Client info to pass to client.
-            credentials (auth_credentials.credentials):
-                Optional. Client credentials to pass to client.
+            client_class (AiPlatformServiceClient):     Required. Class
```

Comment:
It looks like this one needs to be indented with an "Args:" section as it was missing it initially.